test/CMakeLists.txt

```diff
@@ -236,13 +155,13 @@ test_headers(migraphx ${CMAKE_SOURCE_DIR}/src/include/migraphx/*.hpp)
 test_headers(migraphx/ref ${CMAKE_SOURCE_DIR}/src/targets/ref/include/migraphx/ref/*.hpp)
```

Comment:
Missing `HEADERS` ?

Reply:
Nope, `test_headers` doesnt have the argument.